src/meta/mod.rs

```diff
@@ -56,8 +57,46 @@ impl Meta {
             println!("cannot access '{}': {}", path.display(), err);
             return Ok(meta);
         }
+		let mut content = Vec::new();
+
+		if list_hidden_files {
```

Comment:
There needs to be some change to this as the check has to be `all` (a flag that will have to be added in). 

Right now `list_hidden_files` is set to true if `-a` or `-A` is passed as a flag. It should be just `-a`.
The `list_hidden_files` variable comes from `flags.rs` line 68. 

The current one will show `.` and `..` with both `all` and `almost-all`. We need it to be showing when  it is just `all`. You might have to change it there. This was a  placeholder which was set there just until we had this feature added. You might have to add another flag into the mix.

Reply:
> You might have to add another flag into the mix.

Did you mean we have to add a new flag which is 'almost-all' by the 'add another flag into the mix'?

Currently I remove `matches.is_present("almost-all")` on line 68.

Reply:
Right now the condition you are checking now `list_hidden_files` is set to true for both `-a` and `-A` flag, but we should be showing the current and parent directory only if `-a` is used not when `-A` flag is used.

What I meant was that you will have to add a new flag in `flags.rs` to separate them and pass the it to your code.